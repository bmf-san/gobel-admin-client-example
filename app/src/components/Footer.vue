<template>
  <footer class="sticky-footer margin-top-2rem">
    <div class="container">
      <div class="row text-align-center">
        <div class="col">
          <small>gobel-admin-client-example is licensed MIT.</small>
        </div>
      </div>
    </div>
  </footer>
</template>

<script>
export default {
  name: "Footer"
};
</script>
