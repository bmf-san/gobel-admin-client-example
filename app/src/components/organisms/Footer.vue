<template>
  <footer class="sticky-footer margin-top-2rem">
    <div class="container">
      <div class="row text-align-center">
        <div class="col">
          <small>gobel-admin-client-example is licensed MIT.</small>
        </div>
      </div>
    </div>
  </footer>
</template>

<script>
import { defineComponent } from "vue";

export default defineComponent({
  name: "Footer"
});
</script>
