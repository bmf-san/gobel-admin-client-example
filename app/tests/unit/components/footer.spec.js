import { shallowMount } from "@vue/test-utils";
import Footer from "@/components/Footer.vue";

describe("Footer.vue", () => {
  it("init", () => {
    const wrapper = shallowMount(Footer);
    expect(wrapper.isVueInstance).toBeTruthy();
  });

  it("matched snapshot", () => {
    const wrapper = shallowMount(Footer);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
