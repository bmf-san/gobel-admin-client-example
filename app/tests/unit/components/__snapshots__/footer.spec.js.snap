// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue matched snapshot 1`] = `
<footer class="sticky-footer margin-top-2rem">
  <div class="container">
    <div class="row text-align-center">
      <div class="col"><small>gobel-admin-client-example is licensed MIT.</small></div>
    </div>
  </div>
</footer>
`;
